data/nlu/nlu.md

```diff
@@ -5088,7 +5152,6 @@
 - rasax
 - RASAX
 - RASA X
--  rasa x
```

Comment:
um what is this haha

Reply:
I think someone must have saved ` rasa x` instead of `rasa x` as a synonym? seems like a weird thing to do so i imagine it must have been annotation from within rasa x 🤔 anyway i do think it should be deleted though

Reply:
we need to add `x` though https://github.com/RasaHQ/rasa-demo/pull/507/files#diff-d57826d5a178354625db3f820f613afeR3228